### ISSUE TITLE
How to calculate overlapping ratio for my own data?

### ISSUE THREAD

**Issue by daisyranc** [megvii-research/OMNet#12]
Thank you for your excellent work!

I want to test on my own data but I don't know how to calculate the overlapping ratio you mentioned in your paper. Could you give some explanation about it? Thank you so much!

**Comment by hxwork** [comment]
Hi,

Thanks for your concentration. Consider that we have two point clouds A and B, which have been registered by the ground truth transformation. First, for each point in A, we need to calculate its nearest point in B. If the nearest distance is less than the threshold we set. In our paper, this threshold is set to 0.1, and point clouds are normalized into the unit sphere. Then, we need to count the ratio of the number of points that satisfy the requirement to the total number of points as the overlapping ratio.

**Comment by daisyranc** [comment]
@hxwork Thank you for your reply! However, I'm not sure about the overlapping ratio you mentioned in your paper. Because for the loss function (Mask prediction loss) you mentioned alpha is the overlapping ratio of the inputs. How to calculate this 'alpha' if I use my own data?

**Comment by hxwork** [comment]
Actually, you can calculate the overlapping ratio of each pair in your training data and set the average overlapping ratio as alpha.

**Comment by daisyranc** [comment]
@hxwork So how to calculate the overlapping ratio? Could you show the formula of the overlapping ratio? Or this overlapping ratio is the ratio of the number of overlap points to the total number of points?

**Comment by hxwork** [comment]
Your understanding is right, and the calculation manner is as described above.
